### PR TITLE
general fixes

### PR DESCRIPTION
**Pull request by aphkyle**
dont merge yet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ There are 4 config options.
 - `hostname`
 - `port`
 
-The first two are `bool`'s, the third one is a string and the fourth an integer.
+The first two are `bool`'s, the third one is a string and the fourth is an integer.
 
-They can be change by doing the following
+They can be changed by doing the following
```
Comment (thatbirdguythatuknownot) [inline README.md:16]: can add a colon here

File: README.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ There are 4 config options.
 - `hostname`
 - `port`
 
-The first two are `bool`'s, the third one is a string and the fourth an integer.
+The first two are `bool`'s, the third one is a string and the fourth is an integer.
```
Comment (thatbirdguythatuknownot) [inline README.md:14]: > `bool`'s

can be changed into this:
> booleans

**Comment by aphkyle** [comment]
@arnavthedumbass anything to change?

**Comment by arnavthedumbass** [comment]
> @arnavthedumbass anything to change?
> 
> 
> 
> 

nope should i merge now? looks amazing ty <3

**Pull request merged**